mscore/pianoroll/pianolevelschooser.cpp

```diff
@@ -49,23 +49,33 @@ void PianoLevelsChooser::setEventDataPressed()
       PianoLevelsFilter* filter = PianoLevelsFilter::FILTER_LIST[_levelsIndex];
 
       int val = eventValSpinBox->value();
+      int randVal = 0; // initializing randVal as 0
       QList<Note*> noteList = _staff->getNotes();
 
       Score* score = _staff->score();
 
       score->startCmd();
 
+      // initialize random seed:
+      std::srand(eventSeedSpinBox->value());
+
       for (Note* note: noteList) {
             if (!note->selected())
                   continue;
 
+            // Generating a random integer from 0 to eventRandSpinBox->value()
+            if (eventRandSpinBox->value() > 0) { // else it remains 0, as initialized
+                randVal = std::rand()/((RAND_MAX + 1u)/eventRandSpinBox->value());
+            }
```

Comment:
Banner style braces please. Ore none at all, if not needed, like here

Reply:
No need for master, actually. The code style used in this PR exactly matches the code style suggested in https://github.com/musescore/Documentation/blob/master/WorkflowAndGuidelines/CodeGuidelines.md

Reply:
If this PR really is for master only **and** master got changed to that new style

Reply:
I would merge this PR only to master since this is a new feature. Code style in master will be changed by the end of this month.

Reply:
I see